nemo/collections/tts/models/fastpitch.py

```diff
@@ -386,38 +389,62 @@ def list_available_models(cls) -> 'List[PretrainedModelInfo]':
 
         return list_of_models
 
+    ### Export code
+    def input_example(self):
+        """
+        Generates input examples for tracing etc.
+        Returns:
+            A tuple of input examples.
+        """
+        par = next(self.fastpitch.parameters())
+        inp = torch.randint(
+            0, self.fastpitch.encoder.word_emb.num_embeddings, (1, 44), device=par.device, dtype=torch.int64
+        )
+        pitch = torch.randn((1, 44), device=par.device, dtype=torch.float32) * 0.5
+        pace = torch.clamp((torch.randn((1, 44), device=par.device, dtype=torch.float32) + 1) * 0.1, min=0.01)
```

Comment:
Shouldn't pace be a float?

Reply:
No, I'm not looking for a global pace that gets broadcasted to each predicted duration. I want a pace per text token, hence it will be the same size and shape as the tokens

Reply:
In that case, should `pace=1.0` be changed in the `infer()` definition?

Reply:
Doesn't really matter. In python, it accepts both a float or a [B, T] tensor since it'll broadcast.

Reply:
Can you add a comment to specify that both are okay to avoid future confusion?

Reply:
Both are not ok from an export perspective. I require, in onnx, that pace must be a tensor of [B, T] which is what this function `input_example` does. That is why I only define input and output types during `_prepare_for_export`.
I added a `export_teardown` to delete input and output types post export.